agent/netconfig.go

```diff
@@ -22,12 +22,24 @@ import (
 	"net"
 )
 
+const (
+	fullIsolationOption = "full_isolation"
+)
+
+// NetworkRequest specifies messages sent to the
+// agent containing information on how to configure network
+// on its host.
+type NetworkRequest struct {
```

Comment:
I don't either, it sounded better than anything "Config" but I am not 100% happy either. Happy to name it as whatever is suggested.